health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/HelpInfoCard.js

```
@@ -252,12 +366,35 @@ const HelpCard = ({ module, pathVar }) => {
+    //         <TextBlock
+    //           caption={section?.heading?.caption && t(section.heading.caption)}
+    //           header={section?.heading?.header && t(section.heading.header)}
+    //           body={section?.heading?.body && t(section.heading.body)}
+    //           subHeader={section?.heading?.subHeader && t(section.heading.subHeader)}
+    //         />
+    //       )}
+    //       <RenderContent content={section.content} />
+    //     </div>
+    //   ))}
+    // />
+    <AppHelpContent config={config}></AppHelpContent>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)